include/fdreadoutlibs/wib/RAWWIBTriggerPrimitiveProcessor.hpp

```diff
@@ -92,6 +96,22 @@ class RAWWIBTriggerPrimitiveProcessor
     } catch (const ers::Issue& excpt) {
       throw readoutlibs::ResourceQueueError(ERS_HERE, "tp queue", "DefaultRequestHandlerModel", excpt);
     }
+
+/*
```

Comment:
As a general rule, I don't like leaving commented code lying around. Is this something that will be used in the near future, or can it be removed?